src/components/error_tooltip/error_tooltip.ts

```diff
@@ -14,6 +15,10 @@ css/* scss */ `
     border-left: 3px solid red;
     padding: 10px;
   }
+
+  .o-popover:has(.o-error-tooltip) {
+    z-index: ${ComponentsImportance.Popover - 11} !important;
```

Comment:
-11 ??

Reply:
would make sense to have a dedicated entry  in ComponentsImportance, don't you think ?

Reply:
Hey, I looked into your suggestion and tried implementing the same, it works fine but at the same I am unable to fetch the zIndex when I'm doing it via test cases. I will push my current changes in a bit.